### ISSUE TITLE
Fedora: udisksd crashes in poll_with_variant()

### ISSUE THREAD

**Issue by stefwalter** [cockpit-project/cockpit#6119]
https://bugzilla.redhat.com/show_bug.cgi?id=1432743

```
(gdb) bt
#0  poll_with_variant (pid=2128, info=0x0, error=0x55d314772f10, user_data=0x0)
    at udiskslinuxvolumegroupobject.c:633
#1  0x00007f6a4bdf0baf in variant_reader_watch_child (pid=2128, 
    pid@entry=<error reading variable: value has been optimized out>, 
    status=<error reading variable: value has been optimized out>, 
    user_data=0x55d31488de40, 
    user_data@entry=<error reading variable: value has been optimized out>)
    at udiskslvm2daemonutil.c:285
#2  0x00007f6a5e3178a4 in g_child_watch_dispatch (source=<optimized out>, 
    callback=<optimized out>, user_data=<optimized out>) at gmain.c:5323
#3  0x00007f6a5e31ae52 in g_main_dispatch (context=0x55d314683240)
    at gmain.c:3203
#4  g_main_context_dispatch (context=context@entry=0x55d314683240)
    at gmain.c:3856
#5  0x00007f6a5e31b1d0 in g_main_context_iterate (context=0x55d314683240, 
    block=block@entry=1, dispatch=dispatch@entry=1, self=<optimized out>)
    at gmain.c:3929
#6  0x00007f6a5e31b4f2 in g_main_loop_run (loop=0x55d314683770) at gmain.c:4125
#7  0x000055d313648a33 in main (argc=<optimized out>, argv=<optimized out>)
    at main.c:180
```

**Comment by mvollmer** [comment]
See also #5380.  Is Fedora missing the fix, or is the crash coming back despite the fix?

**Comment by mvollmer** [comment]
> Is Fedora missing the fix, or is the crash coming back despite the fix?

Fedora 25 is missing the fix: the fix is on storaged-2.6.3, but fedora 25 only has storaged 2.6.2.

**Comment by cockpituous** [comment]
verify/fedora-25
Ooops, it happened again<hr>
```
Traceback (most recent call last):
  File "check-storage-lvm2", line 116, in testLvm
    "DISK2": True }})
  File "storagelib.py", line 303, in dialog_with_retry
    self.dialog_wait_close()
  File "storagelib.py", line 230, in dialog_wait_close
    self.browser.wait_not_present('#dialog')
  File "testlib.py", line 219, in wait_not_present
    return self.wait_js_func('!ph_is_present', selector)
  File "testlib.py", line 210, in wait_js_func
    return self.phantom.wait("%s(%s)" % (func, ','.join(map(jsquote, args))))
  File "testlib.py", line 738, in <lambda>
    return lambda *args: self._invoke(name, *args)
  File "testlib.py", line 764, in _invoke
    raise Error(res['error'])
Error: timeout
Message recipient disconnected from message bus without replying
```
First occurrence: 2017-03-16T14:18:24.400509 | revision d7fca1363c1b18868eecb70290d6ad8867245a1d, [logs](http://fedorapeople.org/groups/cockpit/logs/pull-6126-d7fca136-verify-fedora-25/log.html)
Times recorded: 1
Latest occurrences:

- 2017-03-16T14:18:24.400509 | revision d7fca1363c1b18868eecb70290d6ad8867245a1d, [logs](http://fedorapeople.org/groups/cockpit/logs/pull-6126-d7fca136-verify-fedora-25/log.html)
